File: src/main/java/org/eightsines/decompiled/A5.java
package org.eightsines.decompiled;

import kotlin.jvm.JvmField;

public final class A5 {
    private final void foo(Item item) {
        item.y = 2;
        System.out.println(item.x + ", " + item.y);
    }

    public static final class Item {
        @JvmField
        public final int x;

        @JvmField
        public int y;

        public Item(int x, int y) {
            this.x = x;
            this.y = y;
        }
    }
}

File: src/main/kotlin/org/eightsines/kotlinperfopt/A5.kt
package org.eightsines.kotlinperfopt

class A5 {
    class Item(@JvmField val x: Int, @JvmField var y: Int)

    private fun foo(item: Item) {
        item.y = 2
        println("${item.x}, ${item.y}")
    }
}
